### PR TITLE
Document SecretExport workaround for carto

### PR DESCRIPTION
**Pull request by acosta11**
[TME-2255]

### CODE REVIEW

**Comment by HenryBorys** [comment]
@tcdowney @emalm Do either of you want to review before I merge and work on this? Please let me know. Thanks!

**Review by emalm — APPROVED** [review]
Looks good to me for content!

**Review by tcdowney — APPROVED** [review]

**Comment by HenryBorys** [comment]
Thanks all!

File: install.md [hunk 1]
```diff
@@ -227,6 +227,43 @@ To configure the installation settings:
      toNamespace: cf
    EOF
    ```
+ 
+If using the experimental Choreographer integration, this SecretExport will also need to grant access to the secret in every cf space namespace. To do so, either:
```
Comment (alexbarbato) [inline install.md:231]: Choreographer should be Cartographer no?

**Comment by HenryBorys** [comment]
@acosta11 @emalm @tcdowney Is this only for main or also v1.1 and v1.0?

File: install.md (hunk 1 above)
Comment (matt-royal) [reply install.md:231]: We appear to use both names, but Choreographer is used more consistently when talking about the integration.

**Pull request merged**